os/arch/arm/src/common/up_allocateheap.c

```diff
@@ -205,15 +218,58 @@ void up_allocate_kheap(FAR void **heap_start, size_t *heap_size)
 }
 #endif
 
+/****************************************************************************
+ * Name: up_addmultiheap
+ ****************************************************************************/
+#if CONFIG_MM_NHEAPS > 1
```

Comment:
#ifdef usage is recommended.
Please check all the #if 's

Reply:
We should check not only that CONFIG_MM_NHEAPS is defined, but also the value of this config.
So I think `#if CONFIG_MM_NHEAPS > 1` is correct.

Reply:
ahh yes